functions/src/import/index.ts

```diff
@@ -45,16 +46,15 @@ const sentDataToAlgolia = (data: any[]) => {
     });
 };
 
-const retrieveDataFromFirestore = async () => {
+const BATCH_MAX_SIZE = 9437184;
+const processQuery = async querySnapshot => {
   let records: any[] = [];
-  const querySnapshot = await database.collection(config.collectionPath).get();
-  const BATCH_MAX_SIZE = 9437184;
-  querySnapshot.forEach((docSnapshot) => {
-    // Capture the record and add to records array for later push to Algolia.
-    // Add in config property to allow up to 100kb if plan allows it.
+  const docs = querySnapshot.docs;
+  const timestamp = Date.now();
+  for (const doc of docs) {
     try {
-      const timestamp = Date.now();
-      records.push(extract(docSnapshot, timestamp));
+      let payload = await extract(doc, timestamp);
```

Comment:
can we add the https://eslint.org/docs/rules/prefer-const, maybe all of https://github.com/algolia/eslint-config-algolia in a future PR?